tasks.py

```diff
@@ -41,12 +41,17 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_chatops": {
-            "nautobot_ver": "1.0.1",
+            "nautobot_ver": "1.1.6",
             "project_name": "nautobot-chatops",
             "python_ver": "3.6",
             "local": False,
             "compose_dir": os.path.join(os.path.dirname(__file__), "development"),
-            "compose_files": ["docker-compose.requirements.yml", "docker-compose.base.yml", "docker-compose.dev.yml"],
+            "compose_files": [
+                "docker-compose.requirements.yml",
+                # "docker-compose.celery.yml",
```

Comment:
If 1.1 is now the default nautobot version, don't we need to include celery by default?

Reply:
It will be. I'm going to do the update to nautobot dev environment in a separate PR. This is a better method to have the Celery components commented out but in the code. It is just easier then to get it up and running.

Reply:
This is just an easier method to handle converting over to using 1.1 or later. When Nautobot 1.3 comes out we will make the updates to go to Python3.7 and update to the newer methods.